extensions/cornerstone-3d/src/utils/dicomRTAnnotationExport/RTStructureSet/measurements/AnnotationToPointData.js

```diff
@@ -0,0 +1,58 @@
+import RectangleROIStartEndThreshold from './RectangleROIStartEndThreshold';
+
+function validateAnnotation(annotation) {
+  if (!annotation?.data) {
+    throw new Error('Tool data is empty');
+  }
+
+  if (!annotation.metadata || annotation.metadata.referenceImageId) {
+    throw new Error('Tool data is not associated with any imageId');
+  }
+}
+
+class AnnotationToPointData {
+  constructor() {}
+
+  static convert(annotation, index, metadataProvider) {
+    validateAnnotation(annotation);
+
+    const { toolName } = annotation.metadata;
+    const toolClass = AnnotationToPointData.TOOL_NAMES[toolName];
+
+    if (!toolClass) {
+      throw new Error(
+        `Unknown tool type: ${toolName}, cannot convert to RTSSReport`
```

Comment:
In general I think this whole thing is honestly a bit too generic. For most annotations, SR makes a lot more sense. RTSTRUCT is going to be extremely specific (points, open/closed 2D/3D contours).